src/TestInfrastructure/VsObjectModelMocks/SolutionMock.cs

```diff
@@ -515,12 +515,15 @@ public IEnumerable<ProjectMock> Projects
             }
         }
 
-        public ProjectMock AddOrGetProject(string projectFile, bool isLoaded = true)
+        public ProjectMock AddOrGetProject(string projectFile, bool isLoaded = true, bool isLegacy = true)
```

Comment:
By default assume we're using using the legacy project system (to minimise the amount of churn required to the existing unit tests).